ERCS/ERC-271.md

```diff
@@ -0,0 +1,93 @@
+---
+eip: 271
```

Comment:
```suggestion
eip: 8055
```
Assigning next sequential EIP/ERC/RIP number.
Numbers are assigned by editors & associates.  You don't get to pick your own.

Please also update the filename.